command/issue.go

```diff
@@ -65,7 +65,17 @@ var issueCreateCmd = &cobra.Command{
 var issueListCmd = &cobra.Command{
 	Use:   "list",
 	Short: "List and filter issues in this repository",
-	RunE:  issueList,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) > 0 {
+			return fmt.Errorf("unknown command %q for %q. Please include whitespace separated issue names within quotes if necessary", args[0], cmd.CommandPath())
```

Comment:
It's not clear to me what "whitespace-separated issue names" are. Do issues have names?

How about a message like this:
```
$ gh issue list -l help wanted
unknown argument "wanted" for `gh issue list`: please quote all flag values that contain spaces.
```

Reply:
Looks good. I'll use this.